src/components/data-selector/data-selector.scss

```diff
@@ -3,6 +3,26 @@
 }
 
 .modal {
```

Comment:
Why is this related to webpack?

Reply:
It is related to possibly one of the loaders that cause the order of css class injection to be inconsistent.